### ISSUE TITLE
1.1.0 Microsoft.NETCore.App support?

### ISSUE THREAD

**Issue by rugdealer** [OmniSharp/generator-aspnet#837]
### Short issue description

Just installed, dotnet restore, dotnet build and then tried dotnet run, got this.

```
Project XXXX (.NETCoreApp,Version=v1.0) was previously compiled. Skipping compilation.
The specified framework 'Microsoft.NETCore.App', version '1.0.1' was not found.
  - Check application dependencies and target a framework version installed at:
      /usr/share/dotnet/shared/Microsoft.NETCore.App
  - The following versions are installed:
      1.1.0
  - Alternatively, install the framework version '1.0.1'.
```

### Expected behavior

Expected the project to run with the lastest version of NetCore.

### Actual behavior

Not running.

### Steps to reproduce the behavior

```
yo aspnet
-> Web API Application
dotnet restore
dotnet build
dotnet run

```
----
#### OS version (Win/Mac/\*Nix?)

Ubuntu 14.04 LTS

#### NodeJS version
```
node --version
```

v7.0.0

#### Yeoman version
```
yo --version
```

1.8.5

#### Generator version
NPM 3.*: 

/usr/local/lib
└── generator-aspnet@0.2.4 

```
npm list -g generator-aspnet
```

NPM 2.*:

│ /home/username/Documents/someFolder/XXXX
│ 
└── (empty)
```
npm ls generator-aspnet -ls=0 --long
```

**Comment by sayedihashimi** [comment]
@rugdealer are you asking for `1.1.0` templates that are based on `project.json`?

FYI in VS we are moving from `project.json` to .csproj. In VS2017 RC is the first release with that support. I've opened an issue #846 so that we can update to that content.

I haven't considered updating the `project.json` based templates to `1.1.0`. If there is enough demand for this I think we can consider updating the existing `project.json` based templates and re-release, but I think its best to have the .csproj based version as the default released version once the changes are implemented.

**Comment by sayedihashimi** [comment]
OK I've updated the readme and released this as version `0.2.5`. When we are done with the .csproj support that will be version `0.3.0`. If we need to maintain the `project.json` based content we can continue releasing `0.2.*`.

FYI I've also created a wiki page to catalog this and added a link in the readme https://github.com/OmniSharp/generator-aspnet/wiki/Version-info.

@rugdealer thanks for the contribution, and @peterblazejewicz thanks for the effort as usual. When we started the project the goal was to make available the same templates that VS has. Now with this, F#, and other unique assets we've gone above and beyond on that. It's really great to see the community doing great things and pushing to provide better tools.
